service/controller/v3/arm_templates/virtual_network_setup.json

```diff
@@ -44,7 +47,9 @@
     "masterSubnetID": "[concat(variables('virtualNetworkID'), '/subnets/', variables('masterSubnetName'))]",
 
     "workerSubnetName": "[concat(parameters('virtualNetworkName'), '-WorkerSubnet')]",
-    "workerSubnetID": "[concat(variables('virtualNetworkID'), '/subnets/', variables('workerSubnetName'))]"
+    "workerSubnetID": "[concat(variables('virtualNetworkID'), '/subnets/', variables('workerSubnetName'))]",
+
+    "vpnSubnetName": "GatewaySubnet",
```

Comment:
I think we should call it `vnetGatewaySubnetName`. `vpn` only happens to be the type of the subnet if I understand correctly.

Reply:
yes correct